tools/spy-detector/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "spy-detector"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+nng = "1.0.1"
+dashmap = "6.1.0"
+chrono = "0.4.38"
+shared = { path = "../../shared" }
+log = "0.4.22"
```

Comment:
This is also already in `shared`. You can use it with

https://github.com/0xB10C/peer-observer/blob/8a445f71608bfd968c1231fbfa5b5864f06ab53c/tools/websocket/src/main.rs#L9

Reply:
okay